test/components/cards/lCard.spec.ts

```diff
@@ -36,4 +40,30 @@ describe('LCard component', () => {
     await card.vm.$nextTick()
     expect(card.vm.showSlot).toBeTruthy()
   })
+
+  it('check doesnt show ".LCard--shadow" when props true', async () => {
+    card.setProps({hasShadow : true})
+    await card.vm.$nextTick()
+    expect(card.find('.LCard--shadow').exists()).toBe(false)
```

Comment:
Aqui não seria uma boa testar se o componente em si possui a classe, ao invés de testar se você o encontra por essa classe?
Exemplo: Usar .classes().contain('.LCard--shadow') ou então no caso de não conter .classes().not.contain....